ui/src/components/BaseFormView.tsx

```diff
@@ -900,15 +916,21 @@ class BaseFormView extends PureComponent<BaseFormProps, BaseFormState> {
             const newFields = update(prevState, { data: changes });
             const tempState = this.clearAllErrorMsg(newFields);
 
+            const { newState } = getModifiedState(
+                tempState,
+                this.props.mode,
+                this.fieldsWithModifications.filter((entity) => entity.field === field)
+            );
```

Comment:
@vtsvetkov-splunk  please have a double check at this one, just found that we actually got that info regarding which field is changed here, so state mods can happen here, so i deleted componentDidUpdate method